sympy/simplify/simplify.py

```diff
@@ -868,6 +873,14 @@ def _nthroot_solve(p, n, prec):
                 return sol
 
 
+def quantsimp(expr):
+    from sympy.physics.units import Quantity
+    a, b = symbols('a, b')
+    a = 1/expr.args[0]
+    b = expr.args[1]
+    return b.scale_factor/a.scale_factor
```

Comment:
This needs to be a bit more complex unfortunately. I would say that it should check if `expr.is_Mul` and in that case collect `Quantity`'s referring to the same dimension.